pages/dashboard/_layout.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Flex } from "@artsy/palette";
+import { BareHeader } from "../../components/BareHeader";
```

Comment:
Can we update `tsconfig` with a `basePath` so that these paths are absolute?

Reply:
yup